packages/main/src/Menu.ts

```diff
@@ -564,7 +568,9 @@ class Menu extends UI5Element {
 		if (menuItem && menuItem.hasSubmenu) {
 			// create new sub-menu
 			this._createSubMenu(menuItem, opener);
-			this._openItemSubMenu(menuItem, opener);
+			setTimeout(() => {
```

Comment:
As we discussed in our conversation, I suggest we change the implementation to async/await.